app/views/categories/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= render "shared/nav" %>
+
+<div class="categories">
+  <% if @categories.empty? %>
+  <div class="get-started">
+    <h3 class="started">Get Started</h3>
+    <span class="div">Add your first budget by clicking on the button below</span>
+  </div>
+  <% else %>
+
+  <div class="categories-container">
+    <% @categories.each do |category| %>
```

Comment:
- [ ] `Required`
- [ ] Amazing job so far 👏 
- [ ] However,if a register and log in, there data is private to them only.
- [ ] but this case data is seen by all users. 😞 
- [ ] please see screenshot below. and make amends 🥂 

![image](https://github.com/ashmalzahra/Budget/assets/66230306/73445f6f-4496-4ad6-a037-30c944792c71)